urwid/tests/test_vterm.py

```diff
@@ -238,6 +243,58 @@ def test_erase_display(self):
         self.write(r'98765\e[8D\e[1Jx')
         self.expect('   x5a98765')
 
+    def test_scrolling_region_simple_incremental(self):
```

Comment:
this extra verbose tests maybe removed or replace "short" versions.
@wardi which approach is better?